src/main/java/com/zufar/icedlatte/review/converter/ReviewDtoConverter.java

```diff
@@ -0,0 +1,17 @@
+package com.zufar.icedlatte.review.converter;
+
+import com.zufar.icedlatte.openapi.dto.ReviewResponse;
+import com.zufar.icedlatte.review.entity.Review;
+import org.springframework.stereotype.Service;
+
+@Service
+public class ReviewDtoConverter {
+
+    public ReviewResponse toReviewResponse(Review review){
+        var response = new ReviewResponse();
+        response.setReviewId(review.getId().toString());
```

Comment:
Add validation about review.getId() != null
Because against you can receive a exception and you will not catch it

Reply:
This scenario is unreachable in my opinion as `id` is always not null. Also, I think we should set up a global handler for `RuntimeException` rather than create custom exception wrappers for such cases.